### PR TITLE
Dynamic min and max pasv ports

### PR DESCRIPTION
**Pull request by dehy**
Because exporting all 101 ports can be too much

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,12 @@ The simplest case is to specify user name and password when starting
 the container:
 
 ```
-docker run -p 11021:21 -it --rm -e FTP_USER=scott -e FTP_PASS=tiger -e HOST=localhost mcreations/openwrt-ftp
+docker run -it --rm \
```
Comment (darabi) [inline README.md]: Would you please leave the simplest case in the docs as it was before and _add_ the possibility to specify the PASV ports?

File: README.md [hunk 2]
```diff
@@ -10,7 +10,12 @@ The simplest case is to specify user name and password when starting
...
+            -p 11021:21 \
+            -p 65100:65100 -p 65101:65101 -p 65102:65102 -p 65103:65103 -p 65104:65104 \
```
Comment (darabi) [inline README.md]: The ports which are published start at 65100, but the value of PASV_MIN_PORT is 65000.

Is this a typo or do I misunderstand?

**Comment by darabi** [comment]
Sorry for the late reply and thank you for the very sensible extension.

I just have marked two issues inline. Should you not have the time (or not agree) to fix them, I can do that after a merge.

So please let me know.

Thanks again

Kambiz

File: README.md (hunk 2 above)
Comment (dehy) [inline README.md]: This is a typo, I'll corrrect it

File: README.md (hunk 1 above)
Comment (dehy) [inline README.md]: deal

**Comment by dehy** [comment]
Here it is!

**Pull request merged**